specific/cmo_sale/models/sale.py

```diff
@@ -93,11 +98,11 @@ def _compute_project_number(self):
 
     @api.model
     def _default_covenant(self):
-        convenants = self.env['sale.convenant.description'].search([
+        covenants = self.env['sale.covenant.description'].search([
```

Comment:
This looks strange. Are we going to have only 1 convenant.description in the future? If so, why do we need it?